webapp/src/components/conference/conference.tsx

```diff
@@ -0,0 +1,323 @@
+                        className='icon icon-arrow-left'
+                        aria-label={'Move up'}
+                    />}
+                {!this.state.minimized &&
+                    <i
+                        onClick={this.minimize}
+                        className='icon icon-arrow-collapse'
+                        aria-label={'Minimize'}
+                    />}
+                {this.state.minimized &&
+                    <i
+                        onClick={this.maximize}
+                        className='icon icon-arrow-expand'
+                        aria-label={'Maximize'}
+                    />}
+                <a
+                    style={{color: 'white'}}
+                    onClick={this.close}
+                    target='_blank'
+                    rel='noopener noreferrer'
+                    href={meetingLink}
+                >
+                    <i
+                        style={{transform: 'rotate(135deg)', display: 'inline-block'}}
+                        className='icon icon-arrow-left'
+                        aria-label={'Open in new tab'}
+                    />
+                </a>
+                <i
+                    onClick={this.close}
+                    className='icon icon-close'
+                    aria-label={'Close'}
+                />
+            </div>);
+    }
+
+    render() {
+        if (this.props.post === null) {
+            return null;
+        }
+        const vw = this.getViewportWidth();
+        const width = this.state.minimized ? MINIMIZED_WIDTH : vw;
+        const vh = this.getViewportHeight();
+        const height = this.state.minimized ? MINIMIZED_HEIGHT : vh;
+        const style = getStyle(height, width, this.state.position);
+        return (
+            <React.Fragment>
+                <div
+                    id='jitsiMeet'
+                    style={style.jitsiMeetContainer}
+                />
+
+                {!this.state.minimized &&
+                    <div style={style.background}/>}
+
+                {this.state.loading &&
+                    <div style={style.loading}>
+                        <i className='fa fa-spinner fa-fw fa-pulse spinner'/>
+                    </div>}
+
+                {this.renderButtons(style)}
+            </React.Fragment>
+        );
+    }
+}
+
+function getStyle(height: number, width: number, position: 'top' | 'bottom'): {[key: string]: React.CSSProperties} {
+    const backgroundZIndex = 1000;
+    const jitsiZIndex = 1100;
+    const buttonsZIndex = 1200;
+    const loadingZIndex = 1200;
+
+    return {
+        jitsiMeetContainer: {
+            position: 'absolute',
+            height,
+            width,
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            background: '#474747',
+            bottom: position === POSITION_TOP ? '' : '20px',
+            top: position === POSITION_TOP ? '20px' : '',
```

Comment:
Shouldn't we use the `BORDER_SIZE` const?

Reply:
Yes, that makes sense.